src/views/Community/View.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+// import Dashboard from '../Dashboard'
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { Row, Progress } from 'reactstrap';
+import { sendCommunityData } from '../../store/actions/community';
+import Payment from '../../components/Payment';
+import { isAuthUserType } from '../../services/Auth';
+import { Link } from 'react-router-dom';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Link' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)